app/components/statistic-list/bar/component.js

```diff
@@ -0,0 +1,21 @@
+import Component from "@glimmer/component";
+
+export default class StatisticListBar extends Component {
+  get didFinishEffortsInBudget() {
+    return this.args.remaining === 0 && !this.didFinishEffortsOverBudget;
+  }
+
+  get didFinishEffortsOverBudget() {
+    return this.args.value > this.args.goal;
+  }
+
+  get spentEffortsBarColor() {
+    if (this.didFinishEffortsInBudget) {
+      return "strong-success";
+    }
```

Comment:
This is still coloring the bar :green_circle: if the currently spent effort is less than the estimated goal and if not remaining effort is set. But we discussed, this should only get green, if the archived flag is `true`. As we have no archived flag yet, *no report should be green*. Only blue like in the old days. I suggest you already add the check for the `archived` flag. As it's not given yet by the backend it should always be `false` anyways. So we won't have to change anything if it gets enabled in the future. What you think?

Reply:
![image](https://github.com/adfinis/timed-frontend/assets/10029904/947dbf74-adfd-4911-a41e-c50a752b8668)